pkgs/development/libraries/libunwind/default.nix

```
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-SmrsZmmR+0XQiJxErt6K1usQgHHDVU/N/2cfnJR5SXY=";
   };
 
+  # Fix for aarch64 and non-4K pages. Remove once upgraded past 1.6.2.
+  patches = lib.optional stdenv.isAarch64 (fetchpatch {
+    url = "https://github.com/libunwind/libunwind/commit/cf6b9aadd60a85107afe5196251ca2ed198a96d0.patch";
+    sha256 = "sha256-z3Hpg98D4UMmrE/LC596RFcyxRTvDjD4k7llDPfz1NI=";
+  });
```

```suggestion
  patches = [
    (fetchpatch {
      url = "https://github.com/libunwind/libunwind/commit/cf6b9aadd60a85107afe5196251ca2ed198a96d0.patch";
      sha256 = "sha256-z3Hpg98D4UMmrE/LC596RFcyxRTvDjD4k7llDPfz1NI=";
    })
  ];
```
Prevent bit rotting.